README.md

```diff
@@ -486,5 +486,5 @@ To report a security issue, please disclose it at [security
 advisory](https://github.com/fmtlib/fmt/security/advisories/new).
 
 This project is maintained by a team of volunteers on a
-reasonable-effort basis. As such, please give us at least 90 days to
```

Comment:
I think hyphen is correct here. I don't remember the exact grammar term for this but basically "reasonable-effort" acts as an adjective. The rest looks good to me, thanks.